qiita_pet/templates/study_description_templates/raw_data_editor_tab.html

```diff
@@ -60,7 +60,19 @@
           </li>
         </ol>
         <br/>
-        <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
+        <table border="0">
+          <tr>
+            <td>
+              <a class="btn btn-primary" onclick="add_prep_template({{raw_data_id}});">Add prep template</a>
+            </td>
+            <td>&nbsp; &nbsp;</td>
+            <td>
+              <a class="btn btn-danger" onclick="if ({{len(available_prep_templates)}} > 0) { alert('This raw data can\'t be erased. First remove the attached prep templates') } else { delete_raw_data({{raw_data_id}}) }">
```

Comment:
The location of this button seems strange but I don't think there's a better place to put it? Could it maybe become a red cross (or the trash bin) by the name of the raw data? i.e. `FASTQ (ID: 7) X`. And only make it visible when the tab is on focus?